Module4_Labs/Lab3_File_System/Cards/41.md

```diff
@@ -4,8 +4,10 @@
 
 <!--concepts={directedGraphs, introToGraphs, useOfGraphs, Depth First Search (DFS), Stack Manipulation}-->
 
+**Step 1: Implementing the DFS Algorithm**
```

Comment:
The steps should all be done by card if that makes sense, they shouldn't continue on through the next cards. This means every card will start will step 1.

Reply:
I resolved this issue.